Gemfile.lock

```diff
@@ -168,10 +168,10 @@ GEM
       rails (>= 4.2.0, < 6.2)
     erubi (1.10.0)
     execjs (2.7.0)
-    ffi (1.14.2)
+    ffi (1.15.0)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    i18n (1.8.8)
+    i18n (1.8.10)
```

Comment:
### [i18n](https://github.com/ruby-i18n/i18n)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.8.8 → 1.8.10

[_[change-log](https://github.com/ruby-i18n/i18n/releases), [source-code](https://github.com/ruby-i18n/i18n)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.8.8...v1.8.10).

These are the first 10 commits:
- (ba8e025) [Use simply require](https://github.com/ruby-i18n/i18n/commit/ba8e025bcadc1daa8605d12560cacbb17e0b9844)
- (0053286) [Run tests on Rails 5.0.x and 5.1.x again](https://github.com/ruby-i18n/i18n/commit/0053286827b85d625bfc0c9dcd39ce4a0c4231de)
- (2a02807) [Run tests on the latest patch version of each Ruby release](https://github.com/ruby-i18n/i18n/commit/2a0280712f62e9e2399eaa1c93be3a9755fb7b0e)
- (075a228) [Rails main only supports Ruby 2.7+ going forward](https://github.com/ruby-i18n/i18n/commit/075a22820e1ab8589128c3d1378c3a136c051171)
- (bb7b689) [Exclude Ruby 3+ with all Rails 5.x versions](https://github.com/ruby-i18n/i18n/commit/bb7b689212a01baec405cc5f027ac4fc6b16fa71)
- (cead3e5) [Prevent overwriting `Hash#except` method present in Ruby 3+](https://github.com/ruby-i18n/i18n/commit/cead3e5143732a303f5eb947ecd8d0f101e46a78)
- (7a885fd) [Merge pull request #550 from taki/update-require](https://github.com/ruby-i18n/i18n/commit/7a885fd958340e855fef26bf9f5905f0f3e865ea)
- (7a262de) [Merge pull request #558 from ruby-i18n/ca-build](https://github.com/ruby-i18n/i18n/commit/7a262de0e6b42e7967f1b580f70812f395e2bc28)
- (3951eee) [Merge pull request #557 from ruby-i18n/ca-except-ruby3](https://github.com/ruby-i18n/i18n/commit/3951eee7b7a51e89b6beda12088c94b5bd8dd82b)
- (aae36c2) [Only attempt to set fallbacks= in tests if method is defined](https://github.com/ruby-i18n/i18n/commit/aae36c2f3e6129720483a176000bf1da9f6c0ffa)


</details>